### ISSUE TITLE
Unable to set WEBP_LIBS?

### ISSUE THREAD

**Issue by OmarShehata** [ImageMagick/ImageMagick#1524]
### Description

I'm trying to compile ImageMagick with a locally built `libwebp`. I can point the `configure` script to the right paths by setting `WEBP_LIBS` and `WEBP_CFLAGS`. However, it seems to be overriding these in favor of the `PKG` config.

### Steps to Reproduce

I am running the command below:

```
./configure WEBP_LIBS="-L/var/app/build/lib -lwebp" WEBP_CFLAGS=-I/var/app/ThirdParty/libwebp/src
```

But it fails to find WebP, and it doesn't put the `-lwebp` flag. Notice `ljpeg` and `lpng12` here, but no WebP:

```
Options used to compile and link:
    PREFIX          = /usr/local
    EXEC-PREFIX     = /usr/local
    VERSION         = 7.0.8
    CC              = gcc
    CFLAGS          = -I/usr/include/libpng12      -fopenmp -g -O2 -Wall -mtune=core2 -fexceptions -pthread -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16
    CPPFLAGS        =   -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16
    PCFLAGS         =
    DEFS            = -DHAVE_CONFIG_H
    LDFLAGS         =
    LIBS            =       -ljpeg   -lpng12                           -lz       -lm
    CXX             = g++
    CXXFLAGS        = -g -O2 -pthread
    FEATURES        = DPC HDRI Cipher OpenMP
    DELEGATES       = mpeg jng jpeg png zlib
```

If I comment out these two lines in the configure script:

```
WEBP_CFLAGS=""
WEBP_LIBS=""
```

Then it works correctly, and I can verify this by running the compiled `identify -list format` and see that WEBP is there.

### System Configuration

- ImageMagick version:  ImageMagick 7.0.8-27 Q16 x86_64 2019-03-22
- Environment (Operating system, version and so on): CentOS release 6.10 (Final), running inside Docker, on Windows 10.

**Comment by urban-warrior** [comment]
Clone libwebp.pc to a new location.  Make your local mods in the clone and then point PKG_CONFIG_PATH to the new path (e.g. export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig).  Now reconfigure/rebuild ImageMagick.  Does that work?

**Comment by OmarShehata** [comment]
Yes! Thank you so much @urban-warrior . That does indeed allow me to easily set the the `WEBP_CFLAGS` and `WEBP_LIBS` !

Although I'm very curious to hear if the way I was setting them as described here was not intended to work or if it's a bug.
